# Generated by Django 2.2.11 on 2020-04-29 21:07

import django.core.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('machine_learning', '0002_auto_20200426_1529'),
    ]

    operations = [
        migrations.CreateModel(
            name='TitanicQuery',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('passanger_class', models.IntegerField(validators=[django.core.validators.MinValueValidator(limit_value=0)])),
                ('passanger_age', models.IntegerField(validators=[django.core.validators.MinValueValidator(limit_value=0)])),
                ('sibling_spouse', models.IntegerField(validators=[django.core.validators.MinValueValidator(limit_value=0)])),
                ('parent_children', models.IntegerField(validators=[django.core.validators.MinValueValidator(limit_value=0)])),
                ('passenger_gender', models.CharField(max_length=200)),
                ('passenger_fare', models.DecimalField(decimal_places=5, max_digits=8)),
            ],
        ),
        migrations.CreateModel(
            name='TitanicPrediction',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('prediction_result', models.IntegerField()),
                ('prediction_probability', models.DecimalField(decimal_places=3, max_digits=5)),
                ('related_query', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='machine_learning.TitanicQuery')),
            ],
        ),
    ]
